tests/test_api.py

```diff
@@ -4,6 +4,7 @@
 import json
 import unittest
 from datetime import datetime, timedelta, timezone
+from unittest import mock
```

Comment:
If you use the decorator like we spoke about yesterday you won't need this import.